pkgs/development/tools/imatix_gsl/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchgit, pcre } :
+
+stdenv.mkDerivation rec {
+    name = "imatix_gsl";
+    version = "4.1";
+    src = fetchgit {
+        url = "https://github.com/imatix/gsl.git";
+	rev = "72192d0d9de17de08d9379602d6482b4e5d402d0";
+	sha256 = "0dyginj8l8f5rgbhlp4f3lhfz2ixdli9192l1bmc5jviklna4phg";
+    };
+
+    buildInputs = [ pcre ];
+
+    buildPhase = ''
```

Comment:
Is it possible to do `cd src` in `preBuild` insead of overriding the entire `buildPhase` here?

Note that overriding a phase means you also miss out on potentially useful hooks/magic.